test/decl/subscript/subscripting.swift

```diff
@@ -310,3 +310,12 @@ struct S4 {
     }
   }
 }
+
+// SR-2575
+struct SR2575 {
+  subscript() -> Int {
+    return 1
+  }
+}
+
+SR2575().subscript() // expected-error{{type 'SR2575' has no member 'subscript'}}
```

Comment:
What do you wan to change here? Offer a fixit to change the call to `subscript` to a proper subscript?